rlkit/envs/wrappers/dm_backend_wrappers.py

```diff
@@ -505,10 +505,17 @@ def render(
         if mode == "human":
             self.renderer.render_to_window()
         elif mode == "rgb_array":
```

Comment:
can you add another if block for using wrist cam, currently your code makes including the wrist cam the default usage.